data/858/982/75/85898275.geojson

```diff
@@ -5,7 +5,7 @@
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
     "geom:area":0.0,
-    "geom:area_square_m":21511309.214026,
+    "geom:area_square_m":0.0,
```

Comment:
What happened to the area?!

Reply:
Perhaps **exportify** was leaving stale areas around, and new exportify doesn't? At least it's consistent with `geom:area` now.

Reply:
Looks like this is now Point geometry that incorrectly had a `geom:area_square_m` value larger than `0.0` (presumably from a bad exportify?). This change exportified and correctly updated the `geom:area_square_m` value to `0.0`... so I think we're good here.